cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/DatabaseServer.java

```diff
@@ -0,0 +1,93 @@
+package com.sequenceiq.cloudbreak.cloud.model;
+
+import com.sequenceiq.cloudbreak.cloud.model.generic.DynamicModel;
+
+import java.util.Map;
+
+public class DatabaseServer extends DynamicModel {
+
+    private final String serverId;
+
+    private final String flavor;
+
+    private final DatabaseEngine engine;
+
+    private final String rootUserName;
+
+    private final String rootPassword;
```

Comment:
Is it ok to store passwords on model objects in CB? I don't know anything about the vault service, but this seems like the kind of thing that might need to go in the vault?

Reply:
I think that this object is not persisted. I actually will be posting another PR with database entities that correspond to these cloud-api objects, and in there, vault is used. So, you are generally correct, but that requirement doesn't apply here.

It's very confusing.